src/utils/validationUtils.ts

```diff
@@ -77,6 +77,9 @@ export const transformNumber = (
 export const isValidPhoneNumber = (phone: string): boolean =>
   /^\+?\(?\d{1,3}\)? ?-?\d{1,3} ?-?\d{3,5} ?-?\d{4}( ?-?\d{3})?/.test(phone);
 
+export const isValidEmail = (email: string): boolean =>
+  /^[A-Za-z0-9_!#$%&'*+/=?`{|}~^.-]+@[A-Za-z0-9.-]+$/i.test(email);
+
```

Comment:
You could add unit tests to isValidEmail function

Reply:
Removed custom validation: [c75cd04](https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/c75cd044c10eb30000572d7391756cb80024568a)